pkg/adapter/mtping/runner.go

```diff
@@ -115,11 +119,13 @@ func (a *cronJobsRunner) Start(stopCh <-chan struct{}) error {
 }
 
 func (a *cronJobsRunner) cronTick(ctx context.Context, event cloudevents.Event) func() {
+	logger := logging.MTLoggerFromContext(ctx)
```

Comment:
This will pull the wrong MTLogger from the context. It will return the last added namespace as the key.